docs/ergo-cli.md

```diff
@@ -3,90 +3,292 @@ id: ergo-cli
 title: Ergo CLI
 ---
 
-To install the Ergo command-line interface (CLI):
+Install the `@accordproject/ergo-cli` npm package to access the Ergo command line interface (CLI). After installation you can use the ergo command and its sub-commands as described below.
 
-```term
-    npm install -g @accordproject/ergo-cli
+To install the Ergo CLI:
 ```
+npm install -g @accordproject/ergo-cli@0.20.0
```

Comment:
I would remove `@0.20.0` from everywhere which always install the latest version. This will make things easier to maintain down the road.